Magic™Editor.user.js

```diff
@@ -3167,8 +3168,8 @@
                 'vertical-align': 'bottom',
                 'margin-left': '5px',
                 'font-size': '12px',
-                'color': '#424242',
-                'background': '#eee',
+                'color': 'var(--white)',
+                'background': 'var(--black-800)',
```

Comment:
The `--black-800` variable is `#e7e8eb` for dark mode, not `#eeeeee`. For light, it's `#242729`. Likewise `--white` is `#2d2d2d` in dark mode and `#ffffff` in light mode, not `#424242`. Are these intended to be one-for-one replacements to bring the values in line with the variable system used in Stacks.css or are they intentionally different as a stylistic choice?

Reply:
These colours govern the background and text colour of the number of replacements shown after the run (next to the magic wand button(:

- [Light theme](https://i.imgur.com/4k3bOyr.png)
- [Dark theme](https://i.imgur.com/xHWK6P3.png)

There were no direct equivalents in SE's colour palette. And choosing too close values like `--black-200` + `--black-400` or `--black-600` + `--black-800` makes the choice look OK in one theme but not the other. I opted for a much starker contrast to satisfy both themes.